e2e-tests/insurance/cypress/e2e/journeys/your-buyer/trading-history/trading-history-page.spec.js

```diff
@@ -164,80 +80,42 @@ context(
           cy.checkText(noRadio().label().first(), FIELD_VALUES.NO);
         });
       });
-
-      it('renders a `save and back` button', () => {
```

Comment:
Does this form no longer have "save and back"?

Reply:
no it does and there is a test for it too?

Reply:
The test has been removed from the test?^